common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/MetricLogger.cs

```diff
@@ -0,0 +1,48 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.Services.Diagnostics {
+    using Microsoft.ApplicationInsights;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.Azure.IIoT.Diagnostics;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Metric logger
+    /// </summary>
+    public sealed class MetricLogger : IMetricLogger  {
+
+        /// <summary>
+        /// Create metric logger
+        /// </summary>
+        /// <param name="config"></param>
+        public MetricLogger(IMetricLoggerConfig config) {
+            telemetryConfiguration = TelemetryConfiguration.Active;
```

Comment:
Use _ for private members.   Check for warnings when you compile, as these should be flagged as naming issues in our build.